src/components/layout/UploadThumb/__specs__/UploadThumb.spec.js

```diff
@@ -0,0 +1,396 @@
+import React from 'react'
+
+import { shallow } from 'enzyme'
+
+import UploadThumb, { computeNewZoom, isImageTooLarge } from '../UploadThumb'
+
+const defaultParams = {
+  current: 0.3,
+  min: 0.1,
+  max: 3,
+  step: 0.01,
+  factor: 10,
+  direction: 1,
+}
+
+describe('src | components | layout | UploadThumb |', () => {
+  let props
+  let image = {
+    lastModified: 1565378546222,
+    lastModifiedDate: 'Fri Aug 09 2019 21:22:26 GMT+0200 (heure d’été d’Europe centrale)',
+    name: 'IMG_4366.jpg',
+    size: 1503804,
+    type: 'image/jpeg',
+    webkitRelativePath: '',
+  }
+
+  beforeEach(() => {
+    props = {
+      collectionName: 'fake',
+      dispatch: jest.fn(),
+      hasExistingImage: false,
+      image,
+      onImageChange: jest.fn(),
+      storeKey: 'image',
+    }
+  })
+
+  it('should match the snapshot', () => {
+    // given
+    props.image = {
+      name: 'IMG_4366.jpg',
+      size: 1503804,
+      type: 'image/jpeg',
+    }
+
+    // when
+    const wrapper = shallow(<UploadThumb {...props} />)
+
+    // then
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  describe('render', () => {
+    it('should render main component with default state', () => {
+      // when
+      const wrapper = shallow(<UploadThumb {...props} />)
+
+      // then
+      expect(wrapper.state('readOnly')).toBe(false)
+      expect(wrapper.state('image')).toStrictEqual(image)
+      expect(wrapper.state('isUploadDisabled')).toBe(false)
+      expect(wrapper.state('zoom')).toBe(1)
+    })
+
+    describe('editor-zone div', () => {
```

Comment:
je ne suis pas fan de ce describe, je préfère l'usage d'un terme plus fonctionnel pour comprendre directement quelle partie du code est testée (sauf s'il s'agit du nom d'une fonction):
```suggestion
    describe('Editor zone', () => {
```

Reply:
👍

Reply:
Mon linter ne veut pas de majuscule dans le describe par contre